src/splitInteger.test.js

```diff
@@ -4,18 +4,26 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(8, 2)).toEqual([4, 4]);
+  expect(splitInteger(10, 2)).toEqual([5, 5]);
```

Comment:
I thought that one "expect..." line is enough. But maybe let the mentors decide.

Reply:
may be.

Reply:
good job, everything works, but really one expect can be enough